Doc/c-api/import.rst

```diff
@@ -310,23 +310,25 @@ Importing Modules
 
 .. c:struct:: _inittab
 
-   Structure describing a single entry in the list of built-in modules.  Each of
-   these structures gives the name and initialization function for a module built
-   into the interpreter.  The name is an ASCII encoded string.  Programs which
+   Structure describing a single entry in the list of built-in modules.
+   Programs which
    embed Python may use an array of these structures in conjunction with
    :c:func:`PyImport_ExtendInittab` to provide additional built-in modules.
-   The structure is defined in :file:`Include/import.h` as::
+   The structure consists of two members:
 
-      struct _inittab {
-          const char *name;           /* ASCII encoded string */
-          PyObject* (*initfunc)(void);
-      };
+   .. c:member:: const char *name
+
+      the module name, as an ASCII encoded string
```

Comment:
Please put an uppercase at the start of the sentence and a dot at the end, same below:

```suggestion
      The module name, as an ASCII encoded string.
```

Reply:
Thank you. I initially wrote this in such way, but then I seen that lowercase without a period is used for `PyMethodDef` members (8edcb30c3f8bdd8099a093146fedbd9b63a3f667), and I change it back. Now I see that descriptions of all other members are written as full sentences, so I think that `PyMethodDef` members should be fixed.